src/MUIDataTable.js

```diff
@@ -268,6 +269,8 @@ class MUIDataTable extends React.Component {
 
         columnOptions = {
           name: column.name,
+          label: column.label,
```

Comment:
Lets change this to 

```
label: column.label ? column.label : column.name
```

Reply:
Also, instead let's make `label`  always defined that way we can remove all these guards we have in the other files checking if label || name.

That will require changing: https://github.com/gregnb/mui-datatables/blob/master/src/MUIDataTable.js#L296
to this:
```
        columnOptions = { ...columnOptions, name: column, label: column };

```